src/addressindex.h

```diff
@@ -68,7 +68,7 @@ struct CMempoolAddressDeltaKey {
     CMempoolAddressDeltaKey(int addressType, uint160 addressHash) {
         type = addressType;
         addressBytes = addressHash;
-        asset = "RTM";
```

Comment:
was this added on other PR to fix getmempooladdresses calls?

Reply:
yes

Reply:
then it getting removed in this one and replace with asset,clear(). is it setting it somewhere else?

Reply:
on this constructor the name should be empty, on the last pr i set it wrong and i only discover that it was wrong too late